src/test/resources/orgs/listing.json

```diff
@@ -0,0 +1,55 @@
+{
+  "@context": [
+    "https://bluebrain.github.io/nexus/contexts/resource.json",
+    "https://bluebrain.github.io/nexus/contexts/admin.json",
+    "https://bluebrain.github.io/nexus/contexts/search.json"
+  ],
+  "_total": 3,
+  "_results": [
+    {
+      "label": "org1",
```

Comment:
`_label`. This is not a problem on listings specifically, just on the `Encoder[Organization]` and `Encoder[Project]`

Reply:
Related to: https://github.com/BlueBrain/nexus/issues/120